src/main/java/kude/models/TaskList.java

```diff
@@ -0,0 +1,35 @@
+package kude.models;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Stream;
+
+public class TaskList implements Serializable {
```

Comment:
Good use of serialization for saving tasks!

Reply:
thanks :) 😸